tests/bootstrap.php

```diff
@@ -19,5 +19,9 @@
 
 function run(Tester\TestCase $testCase)
 {
-	$testCase->run(isset($_SERVER['argv'][1]) ? $_SERVER['argv'][1] : NULL);
```

Comment:
could you please explain this change?

Reply:
Calling run with argument is deprecated. Triggered error itself suggests calling runTest and run methods separately.